public_html/class-mshots.php

```diff
@@ -104,7 +118,7 @@ public function requeue_snapshot() {
 			}
 			memcache_set( $m, $urlkey, 1, 0, 300 );
 
-			$requeue_url = self::renderer . "/queue?url=" . rawurlencode( $this->snapshot_url ) . "&f=" . urlencode( $this->snapshot_file );
+			$requeue_url = self::renderer . "/queue?url=" . rawurlencode( $this->snapshot_url ) . "&f=" . urlencode( $this->snapshot_file ) . '&screen_width=' . $this->screen_width . '&screen_height=' . $this->screen_height;
```

Comment:
We need to make sure we only update this url if these values are non-defualt so as to avoid re-caching all the old images.